frontend/www/js/omegaup/components/contest/AddProblem.vue

```diff
@@ -165,6 +166,15 @@ import { fas } from '@fortawesome/free-solid-svg-icons';
 import { library } from '@fortawesome/fontawesome-svg-core';
 library.add(fas);
 
+interface MappedProblems {
+  [key: string]: {
+    published: string;
+    useLastVersion: boolean;
+    versions: types.ProblemVersion[];
```

Comment:
```suggestion
    problem: types.ProblemsetProblemWithVersions;
```

aquí se puede usar la referencia al problema original. así no hay necesidad de guardar copias del commit ni de las versiones.

Reply:
Listo